apps/src/code-studio/components/progress/StudentSelector.jsx

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {connect} from 'react-redux';
+
+import {queryUserProgress} from '@cdo/apps/code-studio/progressRedux';
+import {updateQueryParam} from '@cdo/apps/code-studio/utils';
+import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
+import i18n from '@cdo/locale';
+
+import style from './unit-overview.module.scss';
+
+const NO_SELECTED_STUDENT_ID = '';
+
+function StudentSelector({
+  //from redux
+  students,
+  selectUser,
+}) {
+  const url_params = new URLSearchParams(window.location.search);
```

Comment:
Because the student selector should only show up on the teacher dashboard, I think we can and should use `react-router-dom`'s `useParams` instead of URLSearchParams.

However, that can be part of cleanup